packages/select/README.md

```diff
@@ -0,0 +1,130 @@
+              fill="currentColor"
+            ></path>
+          </g>
+        </svg>
+      </div>
+      <div></div
+    ></span>
+  </button>
+</div>
+```
+
+## Select Properties
+
+| Prop           | Type                                                | Description                                                                                                   | Default     |
+| -------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ----------- |
+| `children`     | `node`                                              | `<Option />` and `<OptionGroup />` elements.                                                                  |             |
+| `className`    | `string`                                            | Adds a className to the outermost element.                                                                    |             |
+| `darkMode`     | `boolean`                                           | Determines whether or not the component will appear in dark mode.                                             | `false`     |
+| `size`         | `'xsmall'` \| `'small'` \| `'default'` \| `'large'` | Sets the size of the component's elements.                                                                    | `'default'` |
+| `id`           | `string`                                            | id associated with the Select component.                                                                      |             |
+| `label`        | `string`                                            | Text shown in bold above the input element.                                                                   |             |
+| `description`  | `string`                                            | Text that gives more detail about the requirements for the input.                                             |             |
+| `placeholder`  | `string`                                            | The placeholder text shown in the input element when an option is not selected.                               | `'Select'`  |
+| `disabled`     | `boolean`                                           | Disables the component from being edited.                                                                     | `false`     |
+| `value`        | `string`                                            | Sets the `<Option />` that will appear selected and makes the component a controlled component.               | `''`        |
+| `defaultValue` | `string`                                            | Sets the `<Option />` that will appear selected on page load when the component is uncontrolled.              | `''`        |
+| `onChange`     | `function`                                          | A function that gets called when the selected value changes. Receives the value string as the first argument. | `() => {}`  |
```

Comment:
i think we should keep the onChange function signature consistent (so passing up the entire event rather than jus the value, even though the value will be what's used in most use-cases)